### PR TITLE
Adds release notes for 2.13.7

### PR DESCRIPTION
**Pull request by urimandujano**
Exactly what's on the tin!

Adding release notes for 2.13.7 release.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
 
+## Release 2.13.7
+
+### Enabling Pydantic V2
+
+In 2.13.5 we released experimental support for Pydantic V2, which made it co-installable via forced install. In this release, we are enabling co-installation by default which will allow end users to leverage Pydantic V2 in their flows and tasks. Additionally, users can choose to update to Pydantic V2 on their own timeline as we maintain compatability with V1 within flows and tasks.
+
+### Documentation
+
+- Fixes type in release notes - https://github.com/PrefectHQ/prefect/pull/10950
```
Comment (serinamarie) [inline RELEASE-NOTES.md:11]: a typo above ^ 😄 
```suggestion
- Fixes typo in release notes - https://github.com/PrefectHQ/prefect/pull/10950
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.7
+
+### Enabling Pydantic V2
+
+In 2.13.5 we released experimental support for Pydantic V2, which made it co-installable via forced install. In this release, we are enabling co-installation by default which will allow end users to leverage Pydantic V2 in their flows and tasks. Additionally, users can choose to update to Pydantic V2 on their own timeline as we maintain compatability with V1 within flows and tasks.
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:8]:
```suggestion
See implementation details in the following pull request:
- Your PR here
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.7
+
+### Enabling Pydantic V2
+
+In 2.13.5 we released experimental support for Pydantic V2, which made it co-installable via forced install. In this release, we are enabling co-installation by default which will allow end users to leverage Pydantic V2 in their flows and tasks. Additionally, users can choose to update to Pydantic V2 on their own timeline as we maintain compatability with V1 within flows and tasks.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:7]:
```suggestion
In 2.13.5 we released experimental support for Pydantic V2, which made it co-installable via forced install. In this release, we are enabling co-installation by default which will allow you to leverage Pydantic V2 in your flows and tasks. Additionally, you can choose to update to Pydantic V2 on your own timeline as we maintain compatability with V1 within flows and tasks.
```

Comment (serinamarie) [reply RELEASE-NOTES.md:7]: generally we try to make it user-facing

**Review by discdiver — COMMENTED** [review]
Thank you!

File: RELEASE-NOTES.md (hunk 2 above)
Comment (urimandujano) [reply RELEASE-NOTES.md:8]: Just making sure -- the PR linked should be the one removing the pin?

Comment (serinamarie) [reply RELEASE-NOTES.md:8]: 👍

Comment (urimandujano) [reply RELEASE-NOTES.md:11]: Done!

**Review by serinamarie — APPROVED** [review]
Thanks!

**Pull request merged**